src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -6,27 +6,34 @@
 					Saved searches
 				</h1>
 			</kv-grid>
-			<kv-grid class="md:tw-pt-6 lg:tw-pt-8">
-				<div
-					v-for="(search, index) in savedSearches"
-					:key="index"
-				>
-					{{ search.name }}
-				</div>
-			</kv-grid>
+
+			<main class>
```

Comment:
The `main` here can probably be replaced with another element tag as well, and the empty `class` removed.